dist/bin/node

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Custom script to replace node and run with V8 flags that make the execution of the
+# benchmarks more predictable.
+# Depending on the version of node, some flags may be deprecated.
+
+# Retrieve the original path by removing the folder containing CodSpeedHQ/action from the path.
+ORIGINAL_PATH=$(echo "$PATH" | tr ":" "\n" | grep -v "CodSpeedHQ/action" | tr "\n" ":")
+# Save the real node path.
+REAL_NODE_PATH=$(env PATH="$ORIGINAL_PATH" which node)
+
+V8_FLAGS=(
+    "--hash-seed=1"
+    "--random-seed=1"
+    "--no-opt"
+    "--predictable"
+    "--predictable-gc-schedule"
+)
+
+# get node major version, using bash regex
+NODE_MAJOR_VERSION=$($REAL_NODE_PATH --version | sed -E 's/^v([0-9]+)\..*$/\1/')
```

Comment:
what happens if node is not found ? This might be possible if users/action mess up with the PATH

Reply:
maybe just a `set -e` at the top of the script would do it

Reply:
yep, had the same idea 😄 
with it, it will exit early. I will add a debug message

Reply:
ah i didnt see 👍️👍️